### DISCUSSION TITLE
wiki?

### DISCUSSION THREAD

**Discussion by Devanshukoli** [EddieHubCommunity/LinkFree#9909] [General]
I was wondering if we can have **wiki** (that is in github repository) besides **README.md** file. For the *BioDrop* project?

what are your thoughts about this?

**Comment by eddiejaoude** [comment]
I don't think this is required, because our documentation is treated as code in the project and goes via the PR workflow and gets deployed as part of the project https://github.com/EddieHubCommunity/BioDrop/tree/main/pages/docs

**Reply by Devanshukoli** [reply]
Well, That is also the case. Thanks for the feedback :)

**Reply by eddiejaoude** [reply]
no problem
